tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Controllers/OrganisationControllerTests.cs

```diff
@@ -19,6 +22,24 @@ public sealed class OrganisationControllerTests
     {
         private readonly Address _address1 = AddressBuilder.Create().WithLine1("18 Stone Road").Build();
 
+        [Test]
+        public void Constructor_Null_Repository_Throws()
```

Comment:
I think `_NullRepository_` would be clearer.

Reply:
👍